api/queries_repo.go

```diff
@@ -19,7 +19,7 @@ type Repository struct {
 	ID                       string
 	Name                     string
 	NameWithOwner            string
-	Owner                    RepositoryOwner
+	Owner                    RepositoryOwner `json:"owner"`
```

Comment:
The conversion from `Owner` to `owner` should happen automatically so I don't _think_ this is necessary?